packages/website/src/pages/ContentPage/DocPage.tsx

```diff
@@ -72,6 +72,19 @@ function prettify(baseEl: HTMLElement, fileId: string) {
   if (fileId) {
     externallyLinkHeaders(baseEl, fileId);
   }
+
+  highlightComments(baseEl);
+}
+
+function highlightComments(baseEl: HTMLElement) {
+  for (const sup of baseEl.querySelectorAll('sup')) {
+    if (sup.children?.[0].id.startsWith('cmnt_')) {
+      const span = sup.previousElementSibling;
+      if (span && span.nodeName === 'SPAN' && !span.attributes['style']) {
+        span.setAttribute('style', 'background-color: rgb(255, 200, 100)');
```

Comment:
Maybe it would be better if we provide comments inline! Instead of showing some [1]， [2] and jumps after click.

Reply:
yes, that can be done in a future PR. I think it ends up being complicated to handle the case where there are lots of comments.